CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 6.1.0
+
+* Move `base.py` content (file usually located in country packages) to core module `formula_toolbox` so that it can be reused by all countries
+* Use `AbstractScenario` if no custom scenario is defined for a tax and benefit sytem
```

Comment:
Because right now the `AbstractScenario` is not really abstract, and can do the job without being overloaded. 
Without it, the boilerplate country package needs to create its own `Scenario`, which only contains:

```py
from openfisca_core.scenarios import AbstractScenario
class Scenario(AbstractScenario):
     pass
```